lightning/src/ln/functional_tests.rs

```diff
@@ -7895,6 +7914,151 @@ fn test_announce_disable_channels() {
 	}
 }
 
+#[test]
+fn test_priv_forwarding_rejection() {
+	// If we have a private channel with outbound liquidity, and
+	// UserConfig::accept_forwards_to_priv_channels is set to false, we should reject any attempts
+	// to forward through that channel.
+	let chanmon_cfgs = create_chanmon_cfgs(3);
+	let node_cfgs = create_node_cfgs(3, &chanmon_cfgs);
+	let mut no_announce_cfg = test_default_channel_config();
+	no_announce_cfg.channel_options.announced_channel = false;
+	no_announce_cfg.accept_forwards_to_priv_channels = false;
```

Comment:
Guess we shouldn't need this because it's `false` by default?

Reply:
Sure, I just like making the test robust against a change to default values in the future.